src/super_gradients/examples/SG_quickstart_segmentation.ipynb

```diff
@@ -187,13 +187,13 @@
         "                  \"batch_size\": 32,\n",
         "                  \"val_batch_size\":32,\n",
         "                  \"image_mask_transforms_aug\": transforms.Compose(\n",
-        "                      [ColorJitterSeg(brightness=0.5, contrast=0.5, saturation=0.5),\n",
+        "                      [SegColorJitter(brightness=0.5, contrast=0.5, saturation=0.5),\n",
```

Comment:
The other transforms here are also supposed to be renamed.

Reply:
I removed this change anyway because the notebooks were changed since last time